CONTRIBUTING.md

```diff
@@ -17,22 +17,22 @@ your change directly to the repository:
   [pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) into the
   main branch. You may add a description of your contribution into [CREDITS.txt](https://github.com/oneapi-src/oneDPL/blob/main/CREDITS.txt).
 
-# Coding conventions
+# Coding Conventions
 
-clang-format is required except the [test folder](https://github.com/oneapi-src/oneDPL/tree/main/test).
+A clang-format is required, except the [test folder](https://github.com/oneapi-src/oneDPL/tree/main/test).
```

Comment:
```suggestion
clang-format is required, except the [test folder](https://github.com/oneapi-src/oneDPL/tree/main/test).
```
clang-format is a tool, so I suggest to remove the article

Reply:
Thanks! I agree.